### ISSUE TITLE
How to inference by custom tiny yolov1 model?

### ISSUE THREAD

**Issue by sirbato** [AlexeyAB/darknet#2741]

**Comment by sirbato** [comment]
Here is my data file and cfg file:

data:
`classes= 9
train  = exp00017/train.txt
valid  = exp00017/test.txt
names = exp00017/names.txt
backup = exp00017/backup`

cfg:
`[net]
batch=1
subdivisions=1
width=448
height=448
channels=3
momentum=0.9
decay=0.0005

saturation=.75
exposure=.75
hue = .1

learning_rate=0.0005
policy=steps
steps=200,400,600,800,20000,30000
scales=2.5,2,2,2,.1,.1
max_batches = 40000

[convolutional]
batch_normalize=1
filters=16
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=256
activation=leaky

[connected]
output= 931
activation=linear

[detection]
classes=9
coords=4
rescore=1
side=7
num=2
softmax=0
sqrt=1
jitter=.2

object_scale=1
noobject_scale=.5
class_scale=1
coord_scale=5`
